src/app/index.tsx

```diff
@@ -86,9 +87,11 @@ export default (options: Options) => {
   });
 
   const container = () => <Provider store={store}>
-    <Services.Provider value={services}>
-      <navigation.components.NavigationProvider routes={routes} />
-    </Services.Provider>
+    <MessageProvider>
```

Comment:
this is still here

Reply:
I think we ended up on including the `<MessageProvider>` only once on the app root, no? Or was it that we'd wrap every component that uses messages with a `<MessageProvider>`?

The former has fewer `<MessageProvider>` sprinkled in the code while the latter has fewer `<MessageProvider>` sprinkled through the tests